docs/templates/toc.yml

```
@@ -207,6 +207,10 @@
     - name: BigQuery built-in functions
       uid: bigframes.bigquery
     name: bigframes.bigquery
+  - items:
+    - name: GeoSeries
+      uid: bigframes.geoseries
+    name: bigframes.geoseries
```

I think this should be `bigframes.geopandas` to reflect the import path / public module name.

```suggestion
      uid: bigframes.geopandas
    name: bigframes.geopandas
```